tests/basic_env.py

```diff
@@ -41,7 +41,8 @@ class TestDirFixture(object):
     # in tests, we replace foo with bar, so we need to make sure that when we
     # modify a file in our tests, its content length changes.
     BAR_CONTENTS = BAR + "r"
-    CODE = "code.py"
+    CODE_DIR = "code"
+    CODE = "code/code.py"
```

Comment:
Please use os.join.

Also, CODE is used in lots of tests, so moving it like that is dangerous. If you need some file like that in your test - just create it where needed.

Reply:
I wouldn't add any new files to this fixture, it is basically a remnant from pre-pytest times. You have 2 better choices now:
- create whatever you need in test itself,
- create a separate pytest fixture that creates it, if you need the same thing in several tests.